tests/features/step-definitions/steps.js

```diff
@@ -371,6 +371,22 @@ Then(
   }
 )
 
+Then(
+  'verify {string} element on {string} wizard should contains {string}.{string}',
```

Comment:
` 'verify {string} element on {string} wizard should contains {string}.{string}'` - not clear what is the element. In my opinion `dropdown element` will be better.